lib/file_validator.rb

```diff
@@ -84,6 +84,10 @@ def expected_files(scope)
     end
 
     store_base_path = ScopeFileStore.new(@base_path, scope.to_s).path
+    puts "path: #{@base_path}"
+    puts "scope #{scope.to_s}"
```

Comment:
Redundant use of `Object#to_s` in interpolation.<br>Do not write to stdout. Use Rails' logger if you want to log.